bonsai/Data/DataController.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  DataController.swift
+//  bonsai
+//
+//  Created by Vladimir Korolev on 06.12.2021.
+//
+
+import Foundation
+import CoreData
+
+final class DataController: ObservableObject {
+
+    static let sharedInstance = DataController()
+    let container = NSPersistentContainer(name: "Core")
```

Comment:
We can add private access for container if didnt use it outside

Reply:
This one is opened to allow calling 'performBackgroundTask' from outside the class